src/tls/connection.c

```diff
@@ -790,10 +791,18 @@ connection_crypto_init (const char *certfile,
   if (ret != GNUTLS_E_SUCCESS)
     errx (EXIT_FAILURE, "gnutls_certificate_allocate_credentials failed: %s", gnutls_strerror (ret));
 
+  /* check if we have a separate key file */
```

Comment:
This comment isn't true anymore.

Reply:
It is -- gnutls_certificate_set_x509_key_file() fails with `GNUTLS_E_FILE_ERROR` if the key file does not exist.